db/schema.rb

```diff
@@ -757,6 +881,7 @@
           END AS current_age,
       p.gender,
       p.status,
+      p.assigned_facility_id,
```

Comment:
@prabhanshuguptagit I wanted to check this one too. Did this get missed, is it supposed to be here? I dropped and recreated my db locally before running these migrations.

Reply:
That... shouldn't be. This will only be added in #1169, not sure why it shows up here. Also I did a drop and migrate on master, I don't see this in my schema.

Reply:
Actually, looks like it was added in https://github.com/simpledotorg/simple-server/pull/1158, so it _should_ be here 👍

Reply:
Yup, looks like I had a dirty schema somewhere during this process. I've reset it now, thanks 👍